.github/workflows/check_with_mesh_cli.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - name: Install dependencies
         run: |
           pip install -r ./requirements-dev.txt
-          curl -sSfL https://raw.githubusercontent.com/coinbase/mesh-cli/master/scripts/install.sh | sh -s -- -b "$HOME/.local/bin"
+          curl -sSfL https://raw.githubusercontent.com/multiversx/coinbase-mesh-cli/refs/heads/install-rename/scripts/install.sh | sh -s -- -b "$HOME/.local/bin"
```

Comment:
We'll update this once the PR is accepted:
 - https://github.com/coinbase/mesh-cli/pull/423